bot/code_review_bot/report/phabricator.py

```diff
@@ -34,12 +32,8 @@ def __init__(self, configuration={}, *args, **kwargs):
             self.analyzers_skipped, list
         ), "analyzers_skipped must be a list"
 
-        self.mode = configuration.get("mode", MODE_COMMENT)
         self.publish_build_errors = configuration.get("publish_build_errors", False)
-        assert self.mode in (MODE_COMMENT, MODE_HARBORMASTER), "Invalid mode"
-        logger.info(
-            "Will publish using", mode=self.mode, build_errors=self.publish_build_errors
-        )
+        logger.info("Will publish using", build_errors=self.publish_build_errors)
```

Comment:
Nit: I guess we can stop logging here, since it's just logging the configuration variable (or change the log message)